package.json

```diff
@@ -138,6 +138,7 @@
     "prepublishOnly": "node scripts/prepublishOnly.js",
     "prettier": "prettier --config .prettierrc.js --write",
     "prettier:check": "prettier \"packages/**/src/**/*.{js,ts,tsx}\" --check --config .prettierrc.js",
+    "setup-project": "node packages/setup-project",
     "setup-repo": "node scripts/setupRepo",
```

Comment:
Currently we use this `setup-repo` script, and we want to keep it that way. The `setup-project` package only needs to be introduced in user's projects, generated by `cwp-template-full` and `cwp-template-cms`.

Reply:
Ok so its still good that I included the "setup-project" call in the main package.json file?

Reply:
Our main repo doesn't need that, as there won't be such a package. That package will be generated for the user's project, and will be defined in the template. You need to add that script to the `cwp-template-full/template/dependencies.json` - that file is converted to `package.json` during `create-webiny-project` execution.